libs/utils/src/lib/helpers/connections.ts

```diff
@@ -58,6 +58,19 @@ export const isSharedManagedConnector = (connectorId: string) => {
   return true;
 };
 
+export const isSharedManagedConnectorFromPApps = (connectorId: string) => {
+  // Note: connectorId format: /providers/Microsoft.PowerApps/apis/{connector}
+  const fields = connectorId.split('/');
```

Comment:
instead of addressing random numbers of arrays, can this be deconstructed?

Reply:
Currently, this is also how we check for our other managed connectors so I'm going to approve and merge, but I agree that we should deconstruct or just use a regex to check the format of the connector